nilearn/decomposition/tests/test_multi_pca.py

```diff
@@ -55,3 +55,46 @@ def test_multi_pca():
 
     # Smoke test to fit with no img
     assert_raises(TypeError, multi_pca.fit)
+
+def test_multi_pca_score():
+    shape = (6, 8, 10, 5)
+    affine = np.eye(4)
+    rng = np.random.RandomState(0)
+
+    # Create a "multi-subject" dataset
+    data = []
+    for i in range(8):
+        this_data = rng.normal(size=shape)
+        # Create fake activation to get non empty mask
+        this_data[2:4, 2:4, 2:4, :] += 10
```

Comment:
I will simplify this, it copied a few lines of code to generate synthetic data. I guess this should be factorized in a synthetic generation function.